docs/usage/user-stories/maintaining-aur-packages-with-renovate.md

```diff
@@ -0,0 +1,172 @@
+
+First, I only want to run this workflow on pull requests targeting the `main` branch.
+
+```yaml
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+    branches:
+      - main
+```
+
+Next, I’m going to need to check out the entire history of the repository, so I can compare the files changed in the latest commit with the Git history.
+
+```yaml
+jobs:
+  updpkgsums:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+```
+
+Getting the package that changed in a pull request requires a little bit of shell magic.
+
+```yaml
+- name: Find updated package
+  run: |
+    #!/usr/bin/env bash
+    set -euxo pipefail
+
+    echo "pkgbuild=$(git diff --name-only origin/main origin/${GITHUB_HEAD_REF} "*PKGBUILD" | head -1 | xargs dirname)" >> $GITHUB_ENV
+```
+
+Now I’ve found the package that changed in the Renovate pull request, I can update the files.
+
+This step in the workflow uses a private GitHub Action that I have in my `aur-packages` repository.
+I’m not going to break it down here, but at its core it runs `updpkgsums` and `makepkg --printsrcinfo > .SRCINFO` with a little extra configuration required to run Arch Linux on GitHub Actions runners.
+You can [check out the full code on GitHub](https://github.com/JamieMagee/aur-packages/tree/main/.github/actions/aur).
+
+```yaml
+- name: Validate package
+  if: ${{ env.pkgbuild != '' }}
+  uses: ./.github/actions/aur
+  with:
+    action: 'updpkgsums'
+    pkgname: ${{ env.pkgbuild }}
+```
+
+Finally, once the `PKGBUILD` and `.SRCINFO` are updated I need to commit that change back to the pull request.
+
+```yaml
+- name: Commit
+  if: ${{ env.pkgbuild != '' }}
+  uses: stefanzweifel/git-auto-commit-action@3ea6ae190baf489ba007f7c92608f33ce20ef04a # v4.16.0
+  with:
+    file_pattern: '*/PKGBUILD */.SRCINFO'
+```
+
+Check out [this pull request for `bicep-bin`](https://github.com/JamieMagee/aur-packages/pull/62) where Renovate opened a pull request, and my GitHub Actions workflow updated the `b2sums` in the `PKGBUILD` and updated the `.SRCINFO`.
+
+But why stop there?
+Let’s talk about publishing.
+
+## Publishing to the AUR
+
+Each AUR package is its own Git repository.
+So to update a package in the AUR, I only need to push a new commit with the updated `PKGBUILD` and `.SRCINFO`.
+Thankfully, [KSXGitHub](https://github.com/KSXGitHub) created the [github-actions-deploy-aur](https://github.com/KSXGitHub/github-actions-deploy-aur) GitHub Action to streamline the whole process.
+
+If I create a new GitHub Actions workflow to publish to the AUR, I can reuse the first two steps from my previous workflow to check out the repository and find the updated package.
+Then all I need to do is to use the [github-actions-deploy-aur](https://github.com/KSXGitHub/github-actions-deploy-aur) GitHub Action:
```

Comment:
```suggestion
Then all I need to do is to use the `github-actions-deploy-aur` GitHub Action:
```

We already link to the action on line 151, so we can probably drop the second link.